packages/react-components/src/components/CaptionsBanner.tsx

```diff
@@ -47,14 +47,19 @@ export interface _CaptionsBannerProps {
    */
   onRenderAvatar?: OnRenderAvatarCallback;
   strings?: _CaptionsBannerStrings;
+  /**
```

Comment:
so this is a nit, but can you fix line 29 comment... seems like banner is spelled with an extra e.